src/components/ListHeader/ListHeader.js

```diff
@@ -1,26 +1,92 @@
-import React from 'react';
+import React, { useRef, useState } from 'react';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faShareAlt } from '@fortawesome/free-solid-svg-icons';
+import { faShareAlt, faCopy, faPencilAlt, faClock, faTrash } from '@fortawesome/free-solid-svg-icons';
 import { ReactComponent as Logo } from '../../static/logo.svg';
 
 import styles from './styles.module.css';
 
-const ListHeader = () => (
-  <div className={styles.listHeader}>
-    <div className={styles.headerRow}>
-      <Logo className={styles.logo} title="Logo" />
-      <FontAwesomeIcon
-        icon={faShareAlt}
-        className={styles.shareIcon}
-        title="Share this list"
-        tabIndex={0}
-      />
-    </div>
-    <div className={styles.headerRow}>
-      <h3>Shopping List</h3>
-      <h4>15 items</h4>
+const ListHeader = () => {
+  const [listDetailsShown, showListDetails] = useState();
+  const listText = useRef(null);
+
+  const shareList = () => {
+    if (navigator.share) {
+      navigator.share({
+        title: 'Share Your List!',
+        url: 'https:thelist.app',
+      });
+    } else {
+      listText.current.select();
+      document.execCommand('copy');
+    }
```

Comment:
I think document.execCommand is a bit of an outdated way of doing things, I think we want:
```js
const url = 'https://thelist.app';
if (navigator.share) {
  navigator.share({
    title: 'Share Your List!',
    url,
  });
} else if (navigator.clipboard) {
  navigator.clipboard.writeText(url);
}
```

Reply:
Gotta love some outdated js! 
Thanks I'll get this changed :)